layouts/partials/nav.html

```diff
@@ -46,22 +45,29 @@ <h2 class="title is-5">{{$printable}}</h2>
         {{- end -}}
         {{- end -}}
       </div>
-      {{ end }}
 
-      {{ if ne .Kind "taxonomy" }}
-      {{ if (isset .Site.Params "sections_right" ) }}
       <div class="nav-right">
-        {{- range $section, $printable := .Site.Params.sections_right -}}
-        {{- if $printable -}}
-        <a class="nav-item" href="{{$section}}">
-          <h2 class="title is-5">{{$printable}}</h2>
-        </a>
-        {{- end -}}
-        {{- end -}}
+        <form onsubmit="google_search(); return false;">
+            <div class="form-group form-group-lg">
+                <input
+                    type="text"
+                    class="form-control"
+                    id="google-search"
+                    placeholder="Search"
+                />
+            </div>
+        </form>
       </div>
-      {{ end }}
-      {{ end }}
```

Comment:
It will not show the `sections_right` page so some modification is needed. Maybe, I can remove the lines below:

```
{{ if ne .Kind "taxonomy" }}
{{ if (isset .Site.Params "sections_right" ) }}
```
AND remove the lines from `config.toml`.

Reply:
I put most of the code back and only removed:

```toml
"/tags/" = "Tags"
```